### PR TITLE
Ensure that change detection notebook hextile error is fixed

### PR DESCRIPTION
**Pull request by BradNeuberg**
For hextile rendering, correctly handle if a subscription only has one geometry entry.

### CODE REVIEW

File: jupyter-notebooks/analytics/user-guide/03_change_detection.ipynb [hunk 1]
```diff
@@ -527,6 +527,16 @@
     "    # Select only the items with selected observed date\n",
     "    month_items = items_gdf.loc[items_gdf['observed'] == observed]\n",
     "    \n",
+    "    # The heatmap rendering requires 2 or more distinct geometries. If there is only one,\n",
+    "    # let's duplicate it and offset the new geometry by a small amount\n",
+    "    if len(month_items) == 1:\n",
+    "        geom = month_items.iloc[0]['geometry']\n",
+    "        new_geom = shapely.affinity.translate(geom, xoff=0.01)\n",
```
Comment (jmmanso) [inline jupyter-notebooks/analytics/user-guide/03_change_detection.ipynb:534]: @BradNeuberg  need to add import shapely

Comment (BradNeuberg) [reply jupyter-notebooks/analytics/user-guide/03_change_detection.ipynb:534]: Thanks! Great point :/

Comment (BradNeuberg) [reply jupyter-notebooks/analytics/user-guide/03_change_detection.ipynb:534]: Actually, it's imported earlier in the notebook: `from shapely.geometry import shape, Polygon`

Comment (BradNeuberg) [reply jupyter-notebooks/analytics/user-guide/03_change_detection.ipynb:535]: The interesting thing is it worked fine without that import, due to some other things that were imported earlier, but for correctness I added `import shapely`.

File: jupyter-notebooks/analytics/user-guide/03_change_detection.ipynb [hunk 2]
```diff
@@ -527,6 +528,16 @@
     "    # Select only the items with selected observed date\n",
     "    month_items = items_gdf.loc[items_gdf['observed'] == observed]\n",
     "    \n",
+    "    # The heatmap rendering requires 2 or more distinct geometries. If there is only one,\n",
```
Comment (strixcuriosus) [inline jupyter-notebooks/analytics/user-guide/03_change_detection.ipynb:531]: you have put this in the `render_items_for_month` for the dynamic map, but it seems like you might have intended to put it in the heatmap `render_items_for_month` below
https://github.com/planetlabs/notebooks/pull/155/files#diff-c99883be981354d381a1b15066d14eb3R598

File: jupyter-notebooks/analytics/user-guide/03_change_detection.ipynb [hunk 3]
```diff
@@ -527,6 +528,16 @@
     "    # Select only the items with selected observed date\n",
     "    month_items = items_gdf.loc[items_gdf['observed'] == observed]\n",
```
Comment (strixcuriosus) [inline jupyter-notebooks/analytics/user-guide/03_change_detection.ipynb:529]: I tried to reproduce your original problem by adding this line
```
month_items = month_items[:1]
```
and trying the map rendering with and without your new code. The "Dynamic Map" that your current code changes works fine with month_items length of 1. The heatmap, however, crashes. See my other comment about https://github.com/planetlabs/notebooks/pull/155/files#r497755572

Comment (BradNeuberg) [reply jupyter-notebooks/analytics/user-guide/03_change_detection.ipynb:529]: I think under the covers the test subscription we were using that originally only had one entry now looks like it has two detections; I'll move the code as indicated.

File: jupyter-notebooks/analytics/user-guide/03_change_detection.ipynb (hunk 2 above)
Comment (BradNeuberg) [reply jupyter-notebooks/analytics/user-guide/03_change_detection.ipynb]: I've moved this to the correct section.

**Pull request merged**